src/components/GetStartedSection/index.tsx

```diff
@@ -0,0 +1,41 @@
+import { Link } from 'gatsby';
+import React from 'react';
+import './GetStartedSection.scss';
+import beginnerGuideIllustration from '../../images/illustrations/beginners-guide-illustration.svg';
+import doMoreWithNodeIllustration from '../../images/illustrations/do-more-illustration.svg';
+
+interface Props {
+  learnLinkText: string;
+  beginnerGuideHeaderText: string;
+  beginnerGuideBodyText: string;
+  doMoreWithNodeHeaderText: string;
+  doMoreWithNodeBodyText: string;
+}
+
+const GetStartedSection = ({
+  learnLinkText,
+  beginnerGuideBodyText,
+  beginnerGuideHeaderText,
+  doMoreWithNodeBodyText,
+  doMoreWithNodeHeaderText,
+}: Props): JSX.Element => (
+  <section className="get-started-section">
+    <div className="cta-container">
+      <div className="cta">
+        <img src={beginnerGuideIllustration} alt="" className="image" />
+        <h2 className="header">{beginnerGuideHeaderText}</h2>
+        <p className="body-text">{beginnerGuideBodyText}</p>
+      </div>
+      <div className="cta">
+        <img src={doMoreWithNodeIllustration} alt="" className="image" />
+        <h2 className="header">{doMoreWithNodeHeaderText}</h2>
+        <p className="body-text">{doMoreWithNodeBodyText}</p>
+      </div>
+    </div>
+    <Link to="/learn" className="btn circular-container">
```

Comment:
The button should not be pink

Reply:
The button is green in light mode and pink in dark mode. That is how it's designed on Figma: https://www.figma.com/file/lOxAGGg5KXb6nwie7zXkz6/NJ---Design-System

Should the button stay green in both modes?

Reply:
IMO I think it should be green in both. We don't have pick buttons anyplace else.

Reply:
Sounds good. I'll remove the pink version when I make the updates to the copy.